Automattic/wp-calypso
# Theme Showcase: Activate themes with the selected style variation

#### Proposed Changes

This PR passes the selected style variation to the endpoint upon theme activation. As result, the activated theme will have the selected global style applied.

![ezgif com-gif-maker](https://user-images.githubusercontent.com/797888/210978618-9c724926-f3d1-4452-803d-d6da414682ca.gif)


#### Testing Instructions



* Head to the logged-in Theme Showcase `/themes/${site_slug}`. If using calypso.live, the flag `themes/showcase-i4/details-and-preview` is required.
* Find a theme with style variation, open the theme preview modal, and select one of the styles in the sidebar.
* Activate the theme and ensure that your site has the theme with the selected style applied. 

**Note**: Atomic sites will need an update to Jetpack to allow the `style_variation_slug` argument in the `/sites/%s/themes/mine` endpoint. See issue https://github.com/Automattic/jetpack/pull/28207. I propose we delay the Atomic site testing until the next Jetpack release containing this change. 

#### Pre-merge Checklist
